src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -418,36 +436,34 @@ AmendmentTableImpl::doVoting (
 
         if (val.isTrusted ())
         {
-            amendmentSet.addVoter ();
+            vote->addVoter ();
             if (val.isFieldPresent (sfAmendments))
             {
-                for (auto const& amendment : val.getFieldV256 (sfAmendments))
-                {
-                    amendmentSet.addVote (amendment);
-                }
+                auto const amendments = val.getFieldV256 (sfAmendments);
+                for (auto const& a : amendments)
+                    vote->addVote (a);
             }
         }
     }
-    int threshold =
-        (amendmentSet.mTrustedValidations * mMajorityFraction + 255) / 256;
 
-    if (m_journal.trace)
-        m_journal.trace <<
-            amendmentSet.mTrustedValidations << " trusted validations, threshold is "
-            << threshold;
+    vote->mThreshold = (vote->mTrustedValidations * mMajorityFraction) / 256;
```

Comment:
Do we have to worry about `vote->mThreshold == 0`? I.e. an amendment passing with zero votes?